src/ui/components/Rating/img/full-star-gold.svg

```diff
@@ -0,0 +1,10 @@
+<svg width="17px" height="16px" viewBox="0 0 17 16" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
I'm not sure what happened here but this seems to be off by one pixel. When I go to the rating screen and hover over a star, it jumps. I tried fixing it in the SVG markup but it didn't work. I think you have to export it again.

![addons-frontend-stars mov](https://user-images.githubusercontent.com/55398/35940633-6f0dc5c6-0c15-11e8-8e11-4177fcbbec2f.gif)

Reply:
This is probably a result of the hollow star being an export from Sketch and the filled-in star being a copy of an svg already in the repo, but with a different fill colour. I didn't see a filled-in star in sketch for me to export. @pwalm Do you know if there is one in there, and if not might you be able to provide me with a set of hollow and filled-in stars that don't have this issue?

Reply:
Here's a filled-in one https://pastebin.mozilla.org/9077566

I just deleted the inner shape using Sketch. Let me know if that doesn't fix it.

Reply:
Thanks @kumar303, that seems to have fixed it.